metadata-editor/app/controllers/EditsApi.scala

```diff
@@ -40,9 +41,11 @@ object EditsApi extends Controller with ArgoHelpers {
   val usageRightsResponse = {
     // FIXME: GuardianWitness should be there but isn't for simplicity;
     // their images can be imported by drag and drop instead
-    // FIXME: Creating new instances? Rubbish ಠ_ಠ.
+    // FIXME: Creating new instances? Rubbish ಠ_ಠ. I can't think of a way
+    // to access the `val`s of the classes though without instantiating them.
     val usageRightsData =
-      List(PrImage(), Handout(), Screengrab(), SocialMedia(), Obituary(), Pool())
+      List(PrImage(), Handout(), Screengrab(), SocialMedia(), Obituary(), Pool(),
```

Comment:
@theefer would you like me to spend more time at making this better?
I don't particularly like it either, but feel I could be building the interface first, so long as the shape of the API is correct.